AgentDiscoveries-Backend/src/main/java/org/softwire/training/api/models/TokenResponseApiModel.java

```diff
@@ -7,13 +7,15 @@
     private int userId;
     private boolean isAdmin;
     private boolean isAgent;
+    private Integer agentId;
```

Comment:
can we make this an int to keep consistency and also stop it from being nullable